### PR TITLE
dont overwrite ingest logs

### PR DESCRIPTION
**Pull request by afred**
@mccalluc?

### CODE REVIEW

**Comment by afred** [comment]
fixing rubocop... grumble.

File: scripts/ingest.rb [hunk 1]
```diff
@@ -1,8 +1,10 @@
 require_relative '../lib/rails_stub'
 require_relative 'lib/pb_core_ingester'
-require 'logger'
+require_relative '../lib/has_logger'
 require 'rake'
 
+require 'pry'
```
Comment (mccalluc) [inline scripts/ingest.rb]: shouldn't be necessary?

File: scripts/lib/pb_core_ingester.rb [hunk 2]
```diff
@@ -4,11 +4,13 @@
 require 'zip'
 require 'set'
 require_relative '../../app/models/validated_pb_core'
-require_relative 'null_logger'
```
Comment (mccalluc) [inline scripts/lib/pb_core_ingester.rb]: can we git rid of null_logger entirely?

File: scripts/ingest.rb [hunk 3]
```diff
@@ -1,8 +1,10 @@
 require_relative '../lib/rails_stub'
 require_relative 'lib/pb_core_ingester'
-require 'logger'
+require_relative '../lib/has_logger'
```
Comment (mccalluc) [inline scripts/ingest.rb]: unless it's needed by the runtime rails code, could this just be 'lib/has_logger'? Compare with the line above.

**Comment by mccalluc** [comment]
Otherwise, looks great.

File: scripts/ingest.rb (hunk 3 above)
Comment (afred) [inline scripts/ingest.rb]: We haven't added `lib` directory to `$LOAD_PATH`, so you still need to `require_relative`, similar to requiring `rails_stub` above.

**Comment by afred** [comment]
@mccalluc removed pry and NullLogger. Will merge once tests pass.

fyi, I think if you want to do a `NullLogger` in the future, you can just do `Logger.new('/dev/null')`. No need for an extra class

**Pull request merged**